neighbors/src/components/AddPost/AddPost.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { addDoc } from 'firebase/firestore';
+
+import { ThemeContext, needHelpPostsData, offerHelpPostsData } from '../../helpers/apiCommunication';
+export const AddPost = () => {
+  const { user } = useContext(ThemeContext); // Zalogowany user na potrzeby testów
+  const [chooseTypeOfPost, setTypeOfPost] = useState('');
+  const [notAvailbeToWritePost, setAvailbeToWritePost] = useState(true);
```

Comment:
Najcześciej przy useState stosujemy konwencję _state_, _setState_, czyli w tym przypadku byłoby to:
`[notAvailbeToWritePost, setNotAvailbeToWritePost]`, ale w tym przypadku dla uproszczenia proponowałbym usunąć _not_ i default state ustawić jako false, tzn:
`const [availbeToWritePost, setAvailbeToWritePost] = useState(false);`